var/spack/repos/builtin/packages/ufs-weather-model/package.py

```diff
@@ -17,58 +19,129 @@ class UfsWeatherModel(CMakePackage):
     url = "https://github.com/ufs-community/ufs-weather-model/archive/refs/tags/ufs-v1.1.0.tar.gz"
     git = "https://github.com/ufs-community/ufs-weather-model.git"
 
-    maintainers = ["t-brown"]
+    maintainers = ["t-brown", "AlexanderRichert-NOAA"]
 
+    version("develop", branch="develop", submodules=True, commit="ea0b6e4")
     version("2.0.0", tag="ufs-v2.0.0", submodules=True)
     version("1.1.0", tag="ufs-v1.1.0", submodules=True)
 
+    variant("mpi", default=True, description="Enable MPI")
     variant(
         "32bit", default=True, description="Enable 32-bit single precision arithmetic in dycore"
     )
-    variant("avx2", default=False, description="Enable AVX2 instructions")
     variant(
-        "ccpp", default=True, description="Enable the Common Community Physics Package (CCPP))"
+        "ccpp_32bit",
+        default=False,
+        description="Enable CCPP_32BIT (single precision arithmetic in slow physics)",
+    )
+    variant("avx2", default=True, description="Enable AVX2 instruction set")
+    variant(
+        "simdmultiarch", default=False, description="Enable multi-target SIMD instruction sets"
+    )
+    variant("debug", default=False, description="Enable DEBUG mode", when="@develop")
+    variant(
```

Comment:
Note that this horrible debug link mpi option needs to go away, it's not portable and not something we should be doing. @aerorahul @DusanJovic-NOAA and I have fought this bad practice for a while, but with moving to spack-stack we should really get rid of it.